scripts/album.js

```diff
@@ -191,12 +193,13 @@ var setupSeekBars = function() {
 };
 
```

Comment:
You'll need to create a filter function: `var filterTimeCode = function(timeInSeconds)` that takes time in seconds and returns time in minutes and seconds.